[
    {
        "author_association": "NONE",
        "body": "https://github.com/ipython/ipython/commit/dcd9dc90aee7e4c5c52ce44c18e7518934790612\r\n\r\nThe code has been already updated, but `7.19.0` did not include this. :(",
        "created_at": "2020-12-25T17:15:20Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751273086",
        "id": 751273086,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTI3MzA4Ng==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 17,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 17,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751273086/reactions"
        },
        "updated_at": "2020-12-25T17:15:20Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751273086",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/10265739?v=4",
            "events_url": "https://api.github.com/users/benwu95/events{/privacy}",
            "followers_url": "https://api.github.com/users/benwu95/followers",
            "following_url": "https://api.github.com/users/benwu95/following{/other_user}",
            "gists_url": "https://api.github.com/users/benwu95/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/benwu95",
            "id": 10265739,
            "login": "benwu95",
            "node_id": "MDQ6VXNlcjEwMjY1NzM5",
            "organizations_url": "https://api.github.com/users/benwu95/orgs",
            "received_events_url": "https://api.github.com/users/benwu95/received_events",
            "repos_url": "https://api.github.com/users/benwu95/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/benwu95/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/benwu95/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/benwu95"
        }
    },
    {
        "author_association": "NONE",
        "body": "As a temporary fix for anyone just trying to get things working again:\r\n\r\n```\r\npip install jedi==0.17.2\r\n```\r\n\r\n----\r\n\r\nIt would be really nice if you could quickly release a 7.19.1. (It's already fixed on master).\r\n\r\nSorry for that. I did not realize that IPython with that fix was not released yet. I usually test IPython completions before doing a Jedi release, but not this time :/. It will probably also not happen in the future anymore, because I'm going to release Jedi 1.0 soon, so this is probably the last time for a long time that you have to deal with deprecations in Jedi.\r\n\r\nStill wish you a Merry Christmas!",
        "created_at": "2020-12-25T17:20:36Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751273584",
        "id": 751273584,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTI3MzU4NA==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 239,
            "-1": 0,
            "confused": 0,
            "eyes": 8,
            "heart": 98,
            "hooray": 45,
            "laugh": 15,
            "rocket": 26,
            "total_count": 431,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751273584/reactions"
        },
        "updated_at": "2020-12-25T17:22:09Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751273584",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/748594?v=4",
            "events_url": "https://api.github.com/users/davidhalter/events{/privacy}",
            "followers_url": "https://api.github.com/users/davidhalter/followers",
            "following_url": "https://api.github.com/users/davidhalter/following{/other_user}",
            "gists_url": "https://api.github.com/users/davidhalter/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/davidhalter",
            "id": 748594,
            "login": "davidhalter",
            "node_id": "MDQ6VXNlcjc0ODU5NA==",
            "organizations_url": "https://api.github.com/users/davidhalter/orgs",
            "received_events_url": "https://api.github.com/users/davidhalter/received_events",
            "repos_url": "https://api.github.com/users/davidhalter/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/davidhalter/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/davidhalter/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/davidhalter"
        }
    },
    {
        "author_association": "NONE",
        "body": "By the way, a `7.19.1` release with the dependency `jedi<0.18.0` would also suffice.",
        "created_at": "2020-12-25T17:26:00Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751274036",
        "id": 751274036,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTI3NDAzNg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 13,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 13,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751274036/reactions"
        },
        "updated_at": "2020-12-25T17:26:00Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751274036",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/748594?v=4",
            "events_url": "https://api.github.com/users/davidhalter/events{/privacy}",
            "followers_url": "https://api.github.com/users/davidhalter/followers",
            "following_url": "https://api.github.com/users/davidhalter/following{/other_user}",
            "gists_url": "https://api.github.com/users/davidhalter/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/davidhalter",
            "id": 748594,
            "login": "davidhalter",
            "node_id": "MDQ6VXNlcjc0ODU5NA==",
            "organizations_url": "https://api.github.com/users/davidhalter/orgs",
            "received_events_url": "https://api.github.com/users/davidhalter/received_events",
            "repos_url": "https://api.github.com/users/davidhalter/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/davidhalter/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/davidhalter/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/davidhalter"
        }
    },
    {
        "author_association": "NONE",
        "body": "@davidhalter Thanks for your solution. it finally work.",
        "created_at": "2020-12-27T04:33:25Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751425126",
        "id": 751425126,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTQyNTEyNg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 1,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 1,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751425126/reactions"
        },
        "updated_at": "2020-12-27T04:33:25Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751425126",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/1206342?v=4",
            "events_url": "https://api.github.com/users/jkryanchou/events{/privacy}",
            "followers_url": "https://api.github.com/users/jkryanchou/followers",
            "following_url": "https://api.github.com/users/jkryanchou/following{/other_user}",
            "gists_url": "https://api.github.com/users/jkryanchou/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/jkryanchou",
            "id": 1206342,
            "login": "jkryanchou",
            "node_id": "MDQ6VXNlcjEyMDYzNDI=",
            "organizations_url": "https://api.github.com/users/jkryanchou/orgs",
            "received_events_url": "https://api.github.com/users/jkryanchou/received_events",
            "repos_url": "https://api.github.com/users/jkryanchou/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/jkryanchou/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/jkryanchou/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/jkryanchou"
        }
    },
    {
        "author_association": "NONE",
        "body": "I had the same problem with ipython and thanks to the solution of @davidhalter , it works again. :+1: ",
        "created_at": "2020-12-27T14:53:49Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751476672",
        "id": 751476672,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTQ3NjY3Mg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 1,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 1,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751476672/reactions"
        },
        "updated_at": "2020-12-27T14:53:49Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751476672",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/2304703?v=4",
            "events_url": "https://api.github.com/users/SamuelDSR/events{/privacy}",
            "followers_url": "https://api.github.com/users/SamuelDSR/followers",
            "following_url": "https://api.github.com/users/SamuelDSR/following{/other_user}",
            "gists_url": "https://api.github.com/users/SamuelDSR/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/SamuelDSR",
            "id": 2304703,
            "login": "SamuelDSR",
            "node_id": "MDQ6VXNlcjIzMDQ3MDM=",
            "organizations_url": "https://api.github.com/users/SamuelDSR/orgs",
            "received_events_url": "https://api.github.com/users/SamuelDSR/received_events",
            "repos_url": "https://api.github.com/users/SamuelDSR/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/SamuelDSR/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/SamuelDSR/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/SamuelDSR"
        }
    },
    {
        "author_association": "NONE",
        "body": "Thank you for opening this issue.  Happy holiday!",
        "created_at": "2020-12-28T07:28:08Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751613487",
        "id": 751613487,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTYxMzQ4Nw==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751613487/reactions"
        },
        "updated_at": "2020-12-28T07:28:08Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751613487",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/1468378?v=4",
            "events_url": "https://api.github.com/users/digglife/events{/privacy}",
            "followers_url": "https://api.github.com/users/digglife/followers",
            "following_url": "https://api.github.com/users/digglife/following{/other_user}",
            "gists_url": "https://api.github.com/users/digglife/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/digglife",
            "id": 1468378,
            "login": "digglife",
            "node_id": "MDQ6VXNlcjE0NjgzNzg=",
            "organizations_url": "https://api.github.com/users/digglife/orgs",
            "received_events_url": "https://api.github.com/users/digglife/received_events",
            "repos_url": "https://api.github.com/users/digglife/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/digglife/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/digglife/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/digglife"
        }
    },
    {
        "author_association": "NONE",
        "body": "Posted a pr to pin the dependency as suggested: #12746",
        "created_at": "2020-12-28T23:57:08Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751899801",
        "id": 751899801,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTg5OTgwMQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 3,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 3,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751899801/reactions"
        },
        "updated_at": "2020-12-28T23:57:08Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751899801",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/25695763?v=4",
            "events_url": "https://api.github.com/users/yurzo/events{/privacy}",
            "followers_url": "https://api.github.com/users/yurzo/followers",
            "following_url": "https://api.github.com/users/yurzo/following{/other_user}",
            "gists_url": "https://api.github.com/users/yurzo/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/yurzo",
            "id": 25695763,
            "login": "yurzo",
            "node_id": "MDQ6VXNlcjI1Njk1NzYz",
            "organizations_url": "https://api.github.com/users/yurzo/orgs",
            "received_events_url": "https://api.github.com/users/yurzo/received_events",
            "repos_url": "https://api.github.com/users/yurzo/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/yurzo/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/yurzo/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/yurzo"
        }
    },
    {
        "author_association": "NONE",
        "body": "Thanks a lot. I thought my shell was broken. Every time when I tried to use tab completion in IPython it crashed. Glad I found a solution.",
        "created_at": "2020-12-29T02:25:19Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751923463",
        "id": 751923463,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTkyMzQ2Mw==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 2,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 2,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751923463/reactions"
        },
        "updated_at": "2020-12-29T02:25:19Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751923463",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/6715707?v=4",
            "events_url": "https://api.github.com/users/bremme/events{/privacy}",
            "followers_url": "https://api.github.com/users/bremme/followers",
            "following_url": "https://api.github.com/users/bremme/following{/other_user}",
            "gists_url": "https://api.github.com/users/bremme/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/bremme",
            "id": 6715707,
            "login": "bremme",
            "node_id": "MDQ6VXNlcjY3MTU3MDc=",
            "organizations_url": "https://api.github.com/users/bremme/orgs",
            "received_events_url": "https://api.github.com/users/bremme/received_events",
            "repos_url": "https://api.github.com/users/bremme/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/bremme/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/bremme/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/bremme"
        }
    },
    {
        "author_association": "NONE",
        "body": "Finally found a solution! thank you @davidhalter \ud83c\udf89 ",
        "created_at": "2020-12-29T05:05:58Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-751947852",
        "id": 751947852,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTk0Nzg1Mg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 1,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 1,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751947852/reactions"
        },
        "updated_at": "2020-12-29T05:05:58Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/751947852",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/37798612?v=4",
            "events_url": "https://api.github.com/users/aprilahijriyan/events{/privacy}",
            "followers_url": "https://api.github.com/users/aprilahijriyan/followers",
            "following_url": "https://api.github.com/users/aprilahijriyan/following{/other_user}",
            "gists_url": "https://api.github.com/users/aprilahijriyan/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/aprilahijriyan",
            "id": 37798612,
            "login": "aprilahijriyan",
            "node_id": "MDQ6VXNlcjM3Nzk4NjEy",
            "organizations_url": "https://api.github.com/users/aprilahijriyan/orgs",
            "received_events_url": "https://api.github.com/users/aprilahijriyan/received_events",
            "repos_url": "https://api.github.com/users/aprilahijriyan/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/aprilahijriyan/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/aprilahijriyan/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/aprilahijriyan"
        }
    },
    {
        "author_association": "NONE",
        "body": "When would the problem be fixed so that every Jedi version is compatible with ipython?",
        "created_at": "2020-12-31T19:47:45Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-753048651",
        "id": 753048651,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MzA0ODY1MQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753048651/reactions"
        },
        "updated_at": "2020-12-31T19:47:45Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753048651",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/3178925?v=4",
            "events_url": "https://api.github.com/users/tik9/events{/privacy}",
            "followers_url": "https://api.github.com/users/tik9/followers",
            "following_url": "https://api.github.com/users/tik9/following{/other_user}",
            "gists_url": "https://api.github.com/users/tik9/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/tik9",
            "id": 3178925,
            "login": "tik9",
            "node_id": "MDQ6VXNlcjMxNzg5MjU=",
            "organizations_url": "https://api.github.com/users/tik9/orgs",
            "received_events_url": "https://api.github.com/users/tik9/received_events",
            "repos_url": "https://api.github.com/users/tik9/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/tik9/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/tik9/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/tik9"
        }
    },
    {
        "author_association": "NONE",
        "body": "@tik9 Does installing from master work for you?\r\n\r\n    pip install git+https://github.com/ipython/ipython\r\n\r\nIf so, whenever ipython releases a new version, it will be fixed for everyone.",
        "created_at": "2020-12-31T23:52:55Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-753227319",
        "id": 753227319,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MzIyNzMxOQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 5,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 5,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753227319/reactions"
        },
        "updated_at": "2020-12-31T23:53:09Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753227319",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/730137?v=4",
            "events_url": "https://api.github.com/users/NeilGirdhar/events{/privacy}",
            "followers_url": "https://api.github.com/users/NeilGirdhar/followers",
            "following_url": "https://api.github.com/users/NeilGirdhar/following{/other_user}",
            "gists_url": "https://api.github.com/users/NeilGirdhar/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/NeilGirdhar",
            "id": 730137,
            "login": "NeilGirdhar",
            "node_id": "MDQ6VXNlcjczMDEzNw==",
            "organizations_url": "https://api.github.com/users/NeilGirdhar/orgs",
            "received_events_url": "https://api.github.com/users/NeilGirdhar/received_events",
            "repos_url": "https://api.github.com/users/NeilGirdhar/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/NeilGirdhar/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/NeilGirdhar/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/NeilGirdhar"
        }
    },
    {
        "author_association": "NONE",
        "body": "@NeilGirdhar , when doing\r\n`pip install git+https://github.com/ipython/ipython`\r\n\r\nIt seems to work, I still have to update to the current Jedi.",
        "created_at": "2021-01-01T11:07:25Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-753302449",
        "id": 753302449,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MzMwMjQ0OQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 3,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 3,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753302449/reactions"
        },
        "updated_at": "2021-01-01T11:08:48Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753302449",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/3178925?v=4",
            "events_url": "https://api.github.com/users/tik9/events{/privacy}",
            "followers_url": "https://api.github.com/users/tik9/followers",
            "following_url": "https://api.github.com/users/tik9/following{/other_user}",
            "gists_url": "https://api.github.com/users/tik9/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/tik9",
            "id": 3178925,
            "login": "tik9",
            "node_id": "MDQ6VXNlcjMxNzg5MjU=",
            "organizations_url": "https://api.github.com/users/tik9/orgs",
            "received_events_url": "https://api.github.com/users/tik9/received_events",
            "repos_url": "https://api.github.com/users/tik9/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/tik9/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/tik9/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/tik9"
        }
    },
    {
        "author_association": "NONE",
        "body": "Got this issue as well. Pinning `jedi==0.17.2` worked for me, thanks @davidhalter.",
        "created_at": "2021-01-03T02:12:41Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-753556986",
        "id": 753556986,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MzU1Njk4Ng==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753556986/reactions"
        },
        "updated_at": "2021-01-03T02:12:41Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753556986",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/104550?v=4",
            "events_url": "https://api.github.com/users/omarish/events{/privacy}",
            "followers_url": "https://api.github.com/users/omarish/followers",
            "following_url": "https://api.github.com/users/omarish/following{/other_user}",
            "gists_url": "https://api.github.com/users/omarish/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/omarish",
            "id": 104550,
            "login": "omarish",
            "node_id": "MDQ6VXNlcjEwNDU1MA==",
            "organizations_url": "https://api.github.com/users/omarish/orgs",
            "received_events_url": "https://api.github.com/users/omarish/received_events",
            "repos_url": "https://api.github.com/users/omarish/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/omarish/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/omarish/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/omarish"
        }
    },
    {
        "author_association": "NONE",
        "body": "> As a temporary fix for anyone just trying to get things working again:\r\n> \r\n> ```\r\n> pip install jedi==0.17.2\r\n> ```\r\n> \r\n> It would be really nice if you could quickly release a 7.19.1. (It's already fixed on master).\r\n> \r\n> Sorry for that. I did not realize that IPython with that fix was not released yet. I usually test IPython completions before doing a Jedi release, but not this time :/. It will probably also not happen in the future anymore, because I'm going to release Jedi 1.0 soon, so this is probably the last time for a long time that you have to deal with deprecations in Jedi.\r\n> \r\n> Still wish you a Merry Christmas!\r\n\r\nThank you for your solution.\r\nI got the same issue and IPython worked after I installed Jedi",
        "created_at": "2021-01-03T13:05:27Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-753614077",
        "id": 753614077,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MzYxNDA3Nw==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753614077/reactions"
        },
        "updated_at": "2021-01-03T13:05:27Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753614077",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/34534823?v=4",
            "events_url": "https://api.github.com/users/stefanschlipfi/events{/privacy}",
            "followers_url": "https://api.github.com/users/stefanschlipfi/followers",
            "following_url": "https://api.github.com/users/stefanschlipfi/following{/other_user}",
            "gists_url": "https://api.github.com/users/stefanschlipfi/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/stefanschlipfi",
            "id": 34534823,
            "login": "stefanschlipfi",
            "node_id": "MDQ6VXNlcjM0NTM0ODIz",
            "organizations_url": "https://api.github.com/users/stefanschlipfi/orgs",
            "received_events_url": "https://api.github.com/users/stefanschlipfi/received_events",
            "repos_url": "https://api.github.com/users/stefanschlipfi/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/stefanschlipfi/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/stefanschlipfi/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/stefanschlipfi"
        }
    },
    {
        "author_association": "NONE",
        "body": "installing 0.17.2 of jedi also worked for me.  Was chasing my tail trying to figure out why it wasn't working in a new virtual environment, glad to have found this!  Hope the fix is out soon.  Luckily this showed up as the top link in Google for me when searching \"ipython init got an unexpected keyword argument 'column'\".",
        "created_at": "2021-01-04T02:03:25Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-753719755",
        "id": 753719755,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1MzcxOTc1NQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 2,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 2,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753719755/reactions"
        },
        "updated_at": "2021-01-04T02:03:25Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/753719755",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/50895411?v=4",
            "events_url": "https://api.github.com/users/nicksama88/events{/privacy}",
            "followers_url": "https://api.github.com/users/nicksama88/followers",
            "following_url": "https://api.github.com/users/nicksama88/following{/other_user}",
            "gists_url": "https://api.github.com/users/nicksama88/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/nicksama88",
            "id": 50895411,
            "login": "nicksama88",
            "node_id": "MDQ6VXNlcjUwODk1NDEx",
            "organizations_url": "https://api.github.com/users/nicksama88/orgs",
            "received_events_url": "https://api.github.com/users/nicksama88/received_events",
            "repos_url": "https://api.github.com/users/nicksama88/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/nicksama88/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/nicksama88/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/nicksama88"
        }
    },
    {
        "author_association": "NONE",
        "body": "This works:\r\n```console\r\n$ pip install 'jedi<0.18'\r\n```",
        "created_at": "2021-01-04T15:30:44Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-754042613",
        "id": 754042613,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1NDA0MjYxMw==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754042613/reactions"
        },
        "updated_at": "2021-01-04T15:30:44Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754042613",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/54780737?v=4",
            "events_url": "https://api.github.com/users/bl-ue/events{/privacy}",
            "followers_url": "https://api.github.com/users/bl-ue/followers",
            "following_url": "https://api.github.com/users/bl-ue/following{/other_user}",
            "gists_url": "https://api.github.com/users/bl-ue/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/bl-ue",
            "id": 54780737,
            "login": "bl-ue",
            "node_id": "MDQ6VXNlcjU0NzgwNzM3",
            "organizations_url": "https://api.github.com/users/bl-ue/orgs",
            "received_events_url": "https://api.github.com/users/bl-ue/received_events",
            "repos_url": "https://api.github.com/users/bl-ue/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/bl-ue/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/bl-ue/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/bl-ue"
        }
    },
    {
        "author_association": "NONE",
        "body": "Thanks @bl-ue\r\n\r\nFor reference, the autocomplete on my Jupyter notebook wasn't working and I was getting this same error on the terminal.\r\n\r\nAnd now it's working as expected after downgrading jedi to 0.17.2 by just executing the command @bl-ue mentioned.",
        "created_at": "2021-01-04T15:54:29Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-754056158",
        "id": 754056158,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1NDA1NjE1OA==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 1,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 1,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754056158/reactions"
        },
        "updated_at": "2021-01-04T15:54:29Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754056158",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/32599041?v=4",
            "events_url": "https://api.github.com/users/ABODFTW/events{/privacy}",
            "followers_url": "https://api.github.com/users/ABODFTW/followers",
            "following_url": "https://api.github.com/users/ABODFTW/following{/other_user}",
            "gists_url": "https://api.github.com/users/ABODFTW/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/ABODFTW",
            "id": 32599041,
            "login": "ABODFTW",
            "node_id": "MDQ6VXNlcjMyNTk5MDQx",
            "organizations_url": "https://api.github.com/users/ABODFTW/orgs",
            "received_events_url": "https://api.github.com/users/ABODFTW/received_events",
            "repos_url": "https://api.github.com/users/ABODFTW/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/ABODFTW/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/ABODFTW/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/ABODFTW"
        }
    },
    {
        "author_association": "NONE",
        "body": "Confirmed that downgrading jedi to `0.17.2` fixed the issue for me as well. Thanks @davidhalter ",
        "created_at": "2021-01-05T13:10:27Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-754626312",
        "id": 754626312,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1NDYyNjMxMg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 3,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 3,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754626312/reactions"
        },
        "updated_at": "2021-01-05T13:10:27Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754626312",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/19918901?v=4",
            "events_url": "https://api.github.com/users/BallsyWalnuts/events{/privacy}",
            "followers_url": "https://api.github.com/users/BallsyWalnuts/followers",
            "following_url": "https://api.github.com/users/BallsyWalnuts/following{/other_user}",
            "gists_url": "https://api.github.com/users/BallsyWalnuts/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/BallsyWalnuts",
            "id": 19918901,
            "login": "BallsyWalnuts",
            "node_id": "MDQ6VXNlcjE5OTE4OTAx",
            "organizations_url": "https://api.github.com/users/BallsyWalnuts/orgs",
            "received_events_url": "https://api.github.com/users/BallsyWalnuts/received_events",
            "repos_url": "https://api.github.com/users/BallsyWalnuts/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/BallsyWalnuts/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/BallsyWalnuts/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/BallsyWalnuts"
        }
    },
    {
        "author_association": "NONE",
        "body": "Hi everyone, I've stumbled on this issue while creating docker images for myself.\r\nIf I understand correctly from [this comment](https://github.com/ipython/ipython/issues/12740#issuecomment-751273086) the problem has already been solved on master, but so far no release including the fix has been issued.\r\nIn order to better organize my own work, may I know when do you plan a new release? Somebody [here](https://github.com/ipython/ipython/issues/12740#issuecomment-751273584) was suggesting to quickly release 7.19.1, including this patch, is it still an option?",
        "created_at": "2021-01-05T18:57:28Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-754832463",
        "id": 754832463,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1NDgzMjQ2Mw==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 1,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 1,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754832463/reactions"
        },
        "updated_at": "2021-01-05T18:59:13Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/754832463",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/7643149?v=4",
            "events_url": "https://api.github.com/users/gabrielebndn/events{/privacy}",
            "followers_url": "https://api.github.com/users/gabrielebndn/followers",
            "following_url": "https://api.github.com/users/gabrielebndn/following{/other_user}",
            "gists_url": "https://api.github.com/users/gabrielebndn/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/gabrielebndn",
            "id": 7643149,
            "login": "gabrielebndn",
            "node_id": "MDQ6VXNlcjc2NDMxNDk=",
            "organizations_url": "https://api.github.com/users/gabrielebndn/orgs",
            "received_events_url": "https://api.github.com/users/gabrielebndn/received_events",
            "repos_url": "https://api.github.com/users/gabrielebndn/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/gabrielebndn/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/gabrielebndn/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/gabrielebndn"
        }
    },
    {
        "author_association": "NONE",
        "body": "> As a temporary fix for anyone just trying to get things working again:\r\n> \r\n> ```\r\n> pip install jedi==0.17.2\r\n> ```\r\n> \r\n> It would be really nice if you could quickly release a 7.19.1. (It's already fixed on master).\r\n> \r\n> Sorry for that. I did not realize that IPython with that fix was not released yet. I usually test IPython completions before doing a Jedi release, but not this time :/. It will probably also not happen in the future anymore, because I'm going to release Jedi 1.0 soon, so this is probably the last time for a long time that you have to deal with deprecations in Jedi.\r\n> \r\n> Still wish you a Merry Christmas!\r\n\r\nthanks a lot",
        "created_at": "2021-01-07T19:06:42Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-756318031",
        "id": 756318031,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1NjMxODAzMQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 4,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 4,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/756318031/reactions"
        },
        "updated_at": "2021-01-07T19:06:42Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/756318031",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/47916989?v=4",
            "events_url": "https://api.github.com/users/HosseinDahaei/events{/privacy}",
            "followers_url": "https://api.github.com/users/HosseinDahaei/followers",
            "following_url": "https://api.github.com/users/HosseinDahaei/following{/other_user}",
            "gists_url": "https://api.github.com/users/HosseinDahaei/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/HosseinDahaei",
            "id": 47916989,
            "login": "HosseinDahaei",
            "node_id": "MDQ6VXNlcjQ3OTE2OTg5",
            "organizations_url": "https://api.github.com/users/HosseinDahaei/orgs",
            "received_events_url": "https://api.github.com/users/HosseinDahaei/received_events",
            "repos_url": "https://api.github.com/users/HosseinDahaei/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/HosseinDahaei/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/HosseinDahaei/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/HosseinDahaei"
        }
    },
    {
        "author_association": "NONE",
        "body": "pip install -U jedi==0.17.2 parso==0.7.1",
        "created_at": "2021-01-09T16:45:50Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-757333211",
        "id": 757333211,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1NzMzMzIxMQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 1,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 1,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/757333211/reactions"
        },
        "updated_at": "2021-01-09T16:45:50Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/757333211",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/6576768?v=4",
            "events_url": "https://api.github.com/users/LuanComputacao/events{/privacy}",
            "followers_url": "https://api.github.com/users/LuanComputacao/followers",
            "following_url": "https://api.github.com/users/LuanComputacao/following{/other_user}",
            "gists_url": "https://api.github.com/users/LuanComputacao/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/LuanComputacao",
            "id": 6576768,
            "login": "LuanComputacao",
            "node_id": "MDQ6VXNlcjY1NzY3Njg=",
            "organizations_url": "https://api.github.com/users/LuanComputacao/orgs",
            "received_events_url": "https://api.github.com/users/LuanComputacao/received_events",
            "repos_url": "https://api.github.com/users/LuanComputacao/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/LuanComputacao/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/LuanComputacao/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/LuanComputacao"
        }
    },
    {
        "author_association": "NONE",
        "body": "I wonder, when you have a dependency (jedi) that is in version 0.x.y (not reached major 1), if it is not wiser to pin the minor as >=0.17,<0.18.",
        "created_at": "2021-01-11T09:12:16Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-757769588",
        "id": 757769588,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1Nzc2OTU4OA==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 2,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 2,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/757769588/reactions"
        },
        "updated_at": "2021-01-11T09:12:16Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/757769588",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/50870910?v=4",
            "events_url": "https://api.github.com/users/mostealth/events{/privacy}",
            "followers_url": "https://api.github.com/users/mostealth/followers",
            "following_url": "https://api.github.com/users/mostealth/following{/other_user}",
            "gists_url": "https://api.github.com/users/mostealth/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/mostealth",
            "id": 50870910,
            "login": "mostealth",
            "node_id": "MDQ6VXNlcjUwODcwOTEw",
            "organizations_url": "https://api.github.com/users/mostealth/orgs",
            "received_events_url": "https://api.github.com/users/mostealth/received_events",
            "repos_url": "https://api.github.com/users/mostealth/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/mostealth/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/mostealth/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/mostealth"
        }
    },
    {
        "author_association": "NONE",
        "body": "FYI https://github.com/ipython/ipython/pull/12751#issuecomment-758065462",
        "created_at": "2021-01-11T17:39:25Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-758110002",
        "id": 758110002,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1ODExMDAwMg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 1,
            "-1": 0,
            "confused": 0,
            "eyes": 1,
            "heart": 1,
            "hooray": 1,
            "laugh": 1,
            "rocket": 1,
            "total_count": 6,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/758110002/reactions"
        },
        "updated_at": "2021-01-11T17:39:25Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/758110002",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/25695763?v=4",
            "events_url": "https://api.github.com/users/yurzo/events{/privacy}",
            "followers_url": "https://api.github.com/users/yurzo/followers",
            "following_url": "https://api.github.com/users/yurzo/following{/other_user}",
            "gists_url": "https://api.github.com/users/yurzo/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/yurzo",
            "id": 25695763,
            "login": "yurzo",
            "node_id": "MDQ6VXNlcjI1Njk1NzYz",
            "organizations_url": "https://api.github.com/users/yurzo/orgs",
            "received_events_url": "https://api.github.com/users/yurzo/received_events",
            "repos_url": "https://api.github.com/users/yurzo/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/yurzo/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/yurzo/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/yurzo"
        }
    },
    {
        "author_association": "NONE",
        "body": "> As a temporary fix for anyone just trying to get things working again:\r\n> \r\n> ```\r\n> pip install jedi==0.17.2\r\n> ```\r\n> \r\n> It would be really nice if you could quickly release a 7.19.1. (It's already fixed on master).\r\n> \r\n> Sorry for that. I did not realize that IPython with that fix was not released yet. I usually test IPython completions before doing a Jedi release, but not this time :/. It will probably also not happen in the future anymore, because I'm going to release Jedi 1.0 soon, so this is probably the last time for a long time that you have to deal with deprecations in Jedi.\r\n> \r\n> Still wish you a Merry Christmas!\r\n\r\nnote: in anaconda, this has to be done in the same kernel you're using from within the jupyter notebook, not the one you're using the `jupyter notebook` command from!\r\n\r\ntook me about 30 minutes to finally figure that one out once i found @davidhalter's pip install tip (i used `conda install jedi==0.17.2` instead of pip).\r\n\r\nthank you all!",
        "created_at": "2021-01-13T23:40:31Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-759827665",
        "id": 759827665,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc1OTgyNzY2NQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 7,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 7,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/759827665/reactions"
        },
        "updated_at": "2021-01-13T23:40:31Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/759827665",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/7075901?v=4",
            "events_url": "https://api.github.com/users/violet4/events{/privacy}",
            "followers_url": "https://api.github.com/users/violet4/followers",
            "following_url": "https://api.github.com/users/violet4/following{/other_user}",
            "gists_url": "https://api.github.com/users/violet4/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/violet4",
            "id": 7075901,
            "login": "violet4",
            "node_id": "MDQ6VXNlcjcwNzU5MDE=",
            "organizations_url": "https://api.github.com/users/violet4/orgs",
            "received_events_url": "https://api.github.com/users/violet4/received_events",
            "repos_url": "https://api.github.com/users/violet4/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/violet4/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/violet4/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/violet4"
        }
    },
    {
        "author_association": "NONE",
        "body": "Can you explain what you mean by \"same kernel ...\" vs \"using the jupyter notebook ...\"   i.e.,  will conda install jedi==0.17.2 from the shell command line do the trick?  (it seems to)",
        "created_at": "2021-01-15T17:24:05Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-761073748",
        "id": 761073748,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc2MTA3Mzc0OA==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/761073748/reactions"
        },
        "updated_at": "2021-01-15T17:24:05Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/761073748",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/15945736?v=4",
            "events_url": "https://api.github.com/users/gapster/events{/privacy}",
            "followers_url": "https://api.github.com/users/gapster/followers",
            "following_url": "https://api.github.com/users/gapster/following{/other_user}",
            "gists_url": "https://api.github.com/users/gapster/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/gapster",
            "id": 15945736,
            "login": "gapster",
            "node_id": "MDQ6VXNlcjE1OTQ1NzM2",
            "organizations_url": "https://api.github.com/users/gapster/orgs",
            "received_events_url": "https://api.github.com/users/gapster/received_events",
            "repos_url": "https://api.github.com/users/gapster/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/gapster/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/gapster/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/gapster"
        }
    },
    {
        "author_association": "NONE",
        "body": "> Can you explain what you mean by \"same kernel ...\" vs \"using the jupyter notebook ...\" i.e., will conda install jedi==0.17.2 from the shell command line do the trick? (it seems to)\r\n\r\nYou can run Jupyter in one env and use it with a kernel from another env. See for example [this SO question](https://stackoverflow.com/questions/53004311/how-to-add-conda-environment-to-jupyter-lab). Jedi version of the latter env matters. If you are not aware of this, you probably use one conda env for jupyter and kernel.",
        "created_at": "2021-01-16T22:26:28Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-761689616",
        "id": 761689616,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc2MTY4OTYxNg==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 1,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 1,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/761689616/reactions"
        },
        "updated_at": "2021-01-16T22:26:28Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/761689616",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/25516935?v=4",
            "events_url": "https://api.github.com/users/marvinbernhardt/events{/privacy}",
            "followers_url": "https://api.github.com/users/marvinbernhardt/followers",
            "following_url": "https://api.github.com/users/marvinbernhardt/following{/other_user}",
            "gists_url": "https://api.github.com/users/marvinbernhardt/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/marvinbernhardt",
            "id": 25516935,
            "login": "marvinbernhardt",
            "node_id": "MDQ6VXNlcjI1NTE2OTM1",
            "organizations_url": "https://api.github.com/users/marvinbernhardt/orgs",
            "received_events_url": "https://api.github.com/users/marvinbernhardt/received_events",
            "repos_url": "https://api.github.com/users/marvinbernhardt/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/marvinbernhardt/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/marvinbernhardt/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/marvinbernhardt"
        }
    },
    {
        "author_association": "NONE",
        "body": "The following works in ipython/jupyter.   I assume it gets jedi out of the way of the built-in completer.   Good enough for me until fix is released.\r\n\r\n%config Completer.use_jedi = False\r\n",
        "created_at": "2021-01-19T18:00:21Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-763018119",
        "id": 763018119,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc2MzAxODExOQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 7,
            "-1": 2,
            "confused": 0,
            "eyes": 1,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 10,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/763018119/reactions"
        },
        "updated_at": "2021-01-19T18:00:21Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/763018119",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/9634782?v=4",
            "events_url": "https://api.github.com/users/dowenk/events{/privacy}",
            "followers_url": "https://api.github.com/users/dowenk/followers",
            "following_url": "https://api.github.com/users/dowenk/following{/other_user}",
            "gists_url": "https://api.github.com/users/dowenk/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/dowenk",
            "id": 9634782,
            "login": "dowenk",
            "node_id": "MDQ6VXNlcjk2MzQ3ODI=",
            "organizations_url": "https://api.github.com/users/dowenk/orgs",
            "received_events_url": "https://api.github.com/users/dowenk/received_events",
            "repos_url": "https://api.github.com/users/dowenk/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/dowenk/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/dowenk/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/dowenk"
        }
    },
    {
        "author_association": "NONE",
        "body": "It does work! \ud83d\udc4d\ud83c\udffb \r\nNice find @dowenk! \ud83d\ude0d ",
        "created_at": "2021-01-20T14:05:31Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-763627280",
        "id": 763627280,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc2MzYyNzI4MA==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 0,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 0,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/763627280/reactions"
        },
        "updated_at": "2021-01-20T14:05:31Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/763627280",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/54780737?v=4",
            "events_url": "https://api.github.com/users/bl-ue/events{/privacy}",
            "followers_url": "https://api.github.com/users/bl-ue/followers",
            "following_url": "https://api.github.com/users/bl-ue/following{/other_user}",
            "gists_url": "https://api.github.com/users/bl-ue/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/bl-ue",
            "id": 54780737,
            "login": "bl-ue",
            "node_id": "MDQ6VXNlcjU0NzgwNzM3",
            "organizations_url": "https://api.github.com/users/bl-ue/orgs",
            "received_events_url": "https://api.github.com/users/bl-ue/received_events",
            "repos_url": "https://api.github.com/users/bl-ue/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/bl-ue/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/bl-ue/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/bl-ue"
        }
    },
    {
        "author_association": "NONE",
        "body": "Because people at this point probably don't scroll up to my comment (https://github.com/ipython/ipython/issues/12740#issuecomment-751273584), this is what you should do temporarily:\r\n\r\n```\r\npip install jedi==0.17.2\r\n```",
        "created_at": "2021-01-20T17:37:46Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-763815141",
        "id": 763815141,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc2MzgxNTE0MQ==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 16,
            "-1": 0,
            "confused": 0,
            "eyes": 2,
            "heart": 4,
            "hooray": 1,
            "laugh": 3,
            "rocket": 4,
            "total_count": 30,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/763815141/reactions"
        },
        "updated_at": "2021-01-20T17:37:46Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/763815141",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/748594?v=4",
            "events_url": "https://api.github.com/users/davidhalter/events{/privacy}",
            "followers_url": "https://api.github.com/users/davidhalter/followers",
            "following_url": "https://api.github.com/users/davidhalter/following{/other_user}",
            "gists_url": "https://api.github.com/users/davidhalter/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/davidhalter",
            "id": 748594,
            "login": "davidhalter",
            "node_id": "MDQ6VXNlcjc0ODU5NA==",
            "organizations_url": "https://api.github.com/users/davidhalter/orgs",
            "received_events_url": "https://api.github.com/users/davidhalter/received_events",
            "repos_url": "https://api.github.com/users/davidhalter/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/davidhalter/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/davidhalter/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/davidhalter"
        }
    },
    {
        "author_association": "NONE",
        "body": "When is the team going to fix this? In just a month hundreds of users have encountered this issue.",
        "created_at": "2021-01-23T21:39:34Z",
        "html_url": "https://github.com/ipython/ipython/issues/12740#issuecomment-766183653",
        "id": 766183653,
        "issue_url": "https://api.github.com/repos/ipython/ipython/issues/12740",
        "node_id": "MDEyOklzc3VlQ29tbWVudDc2NjE4MzY1Mw==",
        "performed_via_github_app": null,
        "reactions": {
            "+1": 2,
            "-1": 0,
            "confused": 0,
            "eyes": 0,
            "heart": 0,
            "hooray": 0,
            "laugh": 0,
            "rocket": 0,
            "total_count": 2,
            "url": "https://api.github.com/repos/ipython/ipython/issues/comments/766183653/reactions"
        },
        "updated_at": "2021-01-23T21:39:34Z",
        "url": "https://api.github.com/repos/ipython/ipython/issues/comments/766183653",
        "user": {
            "avatar_url": "https://avatars.githubusercontent.com/u/54780737?v=4",
            "events_url": "https://api.github.com/users/bl-ue/events{/privacy}",
            "followers_url": "https://api.github.com/users/bl-ue/followers",
            "following_url": "https://api.github.com/users/bl-ue/following{/other_user}",
            "gists_url": "https://api.github.com/users/bl-ue/gists{/gist_id}",
            "gravatar_id": "",
            "html_url": "https://github.com/bl-ue",
            "id": 54780737,
            "login": "bl-ue",
            "node_id": "MDQ6VXNlcjU0NzgwNzM3",
            "organizations_url": "https://api.github.com/users/bl-ue/orgs",
            "received_events_url": "https://api.github.com/users/bl-ue/received_events",
            "repos_url": "https://api.github.com/users/bl-ue/repos",
            "site_admin": false,
            "starred_url": "https://api.github.com/users/bl-ue/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/bl-ue/subscriptions",
            "type": "User",
            "url": "https://api.github.com/users/bl-ue"
        }
    }
]